docs/faq.md

```diff
@@ -3,40 +3,107 @@
 We list some common troubles faced by many users and their corresponding solutions here.
 Feel free to enrich the list if you find any frequent issues and have ways to help others to solve them.
 
+### Installation
+
+- "KeyError: "MaskRCNN: 'RefineRoIHead is not in the models registry'"
+
+    The registry mechanism will be triggered only when the file where the module is located is imported.
+    So you need to import that file somewhere. More details can be found at https://github.com/open-mmlab/mmdetection/issues/5974.
+
 - Compatibility issue between MMCV and MMDetection; "ConvWS is already registered in conv layer"
```

Comment:
We can put this question to the last as this is an old issue when the ops are first migrated to mmcv from mmdet